x-pack/test/api_integration/apis/management/rollup/rollup.test_helpers.js

```diff
@@ -101,23 +101,29 @@ export const registerHelpers = ({ supertest, es }) => {
 
   const deleteIndicesGeneratedByJobs = () => (
     supertest.get(`${API_BASE_PATH}/indices`)
-      .then(({ body }) => {
+      .then(({ status, body }) => {
+
+        if (status !== 200) {
+          throw new Error(`Error fetching rollup indices with error: "${JSON.stringify(body)}"`);
+        }
+
         const index = Object.keys(body);
+
         if (!index.length) {
           return;
         }
-        return es.indices.delete({ index });
+
+        return deleteIndex(index);
       })
   );
 
   const cleanUp = () => (
     Promise.all([
-      deleteAllIndices(),
       stopAllJobs().then(deleteJob),
-      deleteIndicesGeneratedByJobs(),
+      deleteIndicesGeneratedByJobs().then(deleteAllIndicesCreated),
```

Comment:
This was my silly mistake @spalger 😓I was reading the `Promise.all` sequentially instead of in parallel!

Reply:
Glad you tracked it down!